frontend/src/scenes/pipeline/AppsManagement.tsx

```diff
@@ -0,0 +1,394 @@
+                                            }
+                                        >
+                                            <LemonButton
+                                                type="secondary"
+                                                size="small"
+                                                icon={<IconLockOpen />}
+                                                onClick={() => patchPlugin(plugin.id, { is_global: true })}
+                                            >
+                                                Make global
+                                            </LemonButton>
+                                        </Tooltip>
+                                    )}
+                                </>
+                            )
+                        },
+                    },
+                    {
+                        title: 'Uninstall',
+                        width: '10%',
+                        render: function RenderUninstall(_, plugin) {
+                            return (
+                                <Popconfirm
+                                    placement="topLeft"
+                                    title="Are you sure you wish to uninstall this app completely?"
+                                    onConfirm={() => uninstallPlugin(plugin.id)}
+                                    okText="Uninstall"
+                                    cancelText="Cancel"
+                                    className="Plugins__Popconfirm"
+                                >
+                                    <LemonButton
+                                        type="primary"
+                                        status="danger"
+                                        size="small"
+                                        icon={<IconDelete />}
+                                        disabledReason={
+                                            unusedPlugins.includes(plugin.id) ? undefined : 'This app is still in use.'
+                                        }
+                                        data-attr="plugin-uninstall"
+                                    >
+                                        Uninstall
+                                    </LemonButton>
+                                </Popconfirm>
+                            )
+                        },
+                    },
+                ]}
+            />
+        </>
+    )
+}
+
+function OutOfSyncApps(): JSX.Element {
+    const { missingGlobalPlugins, shouldBeGlobalPlugins, shouldNotBeGlobalPlugins } = useValues(appsManagementLogic)
+    if (
+        missingGlobalPlugins.length === 0 &&
+        shouldBeGlobalPlugins.length === 0 &&
+        shouldNotBeGlobalPlugins.length === 0
+    ) {
+        return <></>
+    }
+    return (
+        <>
+            <LemonBanner type="warning"> Apps state is out of sync </LemonBanner>
+            <br />
+            <MissingGlobalPlugins />
```

Comment:
I'm worried this section will be just confusing for hobby users. I think we should omit the whole global apps system on non-Cloud instances

Reply:
good idea, will gate for `isDevOrCloud`